### ISSUE TITLE
Accessibility: More menu on cards not accessible

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#2383]
The icon "..." should not be announced with AT
You can't tab through both items in the pop-up menu or use the voiceover controls "capslock <>" to navigate them both. It's very confusing to try and select items using a keyboard.

Check out the more menu on the "Forms" page to see how it works. 
The accessibility considerations in Figma were modelled after that menu:

![image.png](https://images.zenhubusercontent.com/620427f3846ebb620d78e582/a5911144-cbef-4d7d-b013-306e1dbc2348)

![image.png](https://images.zenhubusercontent.com/620427f3846ebb620d78e582/42755829-9e74-452a-95d2-716191a7fbe1)
